core/block/editor/clipboard/paste.go

```
@@ -310,16 +315,38 @@ func (p *pasteCtrl) removeSelection() {
 	}
 }
 
-func (p *pasteCtrl) processFiles() {
+func (p *pasteCtrl) processFiles() (err error) {
 	p.ps.Iterate(func(b simple.Block) (isContinue bool) {
 		if file := b.Model().GetFile(); file != nil && file.State == model.BlockContentFile_Empty {
+			if strings.HasPrefix(file.Name, base64ImagePrefix) {
+				err = p.handleBase64(b, file)
```

error is not handled, error can be shadowed in the next iteration